cockroachcloud/backups-page.md

```diff
@@ -20,7 +20,7 @@ Cockroach Labs runs [full cluster backups](../{{site.current_cloud_version}}/tak
 </section>
 
 <section class="filter-content" markdown="1" data-scope="dedicated">
-Cockroach Labs runs [full cluster backups](../{{site.current_cloud_version}}/take-full-and-incremental-backups.html#full-backups) daily and [incremental cluster backups](../{{site.current_cloud_version}}/take-full-and-incremental-backups.html#incremental-backups) hourly for every {{ site.data.products.dedicated }} cluster. The full backups are retained for 30 days, while incremental backups are retained for 7 days.
+Cockroach Labs runs [full cluster backups](../{{site.current_cloud_version}}/take-full-and-incremental-backups.html#full-backups) daily and [incremental cluster backups](../{{site.current_cloud_version}}/take-full-and-incremental-backups.html#incremental-backups) hourly for every {{ site.data.products.dedicated }} cluster. The full backups are retained for 30 days, while incremental backups are retained for 7 days. Backups are stored in the same region that a single-region cluster is running in or the primary region of a multi-region cluster.
```

Comment:
Suggest making 'single-region cluster' a link to https://www.cockroachlabs.com/docs/cockroachcloud/plan-your-cluster.html#cluster-configuration (or something better, you know Cloud docs better than me)

Suggest making 'multi-region cluster' a link to https://www.cockroachlabs.com/docs/cockroachcloud/plan-your-cluster.html#multi-region-clusters (or again something better if it exists)